### ISSUE TITLE
Code outdated?

### ISSUE THREAD

**Issue by thebrumby** [dvandal/cryptonote-nodejs-pool#508]
Hello, I am experimenting with a mining pool for ZOD / WXTC and I can't mine either the main coin or child coin. When I use the same config with Cryptonote-NodeJS-pool 1.4.0 then both configs mine the main coin fine but (obviously) ignores the child coin. Both are cn-lite coins.

Is this code outdated? Or I need to try harder to make it work? Thanks

**Comment by muscleman** [comment]
The merge mining code is from campurro repo. If the child coin is a valid sibling it should work.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by thebrumby** [comment]
Hi Muscleman, thanks for the reply, it's much appreciated any help you might give. 

I wasn't very clear in my original question. I tried as a first attempt setting up a pool with the Campurro repository. I entered the details as merged mining = true and put in the main and child wallets and the main and child daemons. I received a lot of errors so tried to simplify things, and set merged mining = false. After tweaking some settings for daemonType and cnBlobType then it successfully  accepted shares for the main coin and found blocks, but after the maturity time elapsed, the blocks became orphaned.

So long story short, after many unsuccessful attempts to make it work with the Campurro repo, I notice the it showed powered by Cryptonote-node-js-pool v1.35 but I noticed on the D Vandall github page there was a v1.40. So I clone that and used my existing config.json and everything worked fine right out the box; accepted shares - found blocks - not orphaned - paid out.

That's why I wonder if I was using an outdated repo or some other issue. In any case, if that is the up-to-date code then I will just try again. 

Many, many thanks. Phil.

**Comment by muscleman** [comment]
I believe the repo is up to date. This is a community project

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
